server/templates/onboarding_page.html

```diff
@@ -11,44 +11,43 @@
 {% block body %}
   <div class="row-fluid">
     <div class="span12" id="onboarding-container">
-
+      <h2>Thanks for signing up!</h2>
```

Comment:
Two exclamation marks in two successive sentences seems a little excessive; I propose:
Thanks for signing up! ... ! -> Thanks for signing up. ... !